service/hook/github/metrics.go

```diff
@@ -12,7 +13,12 @@ import (
 
 // GatherMetrics ...
 func (hp HookProvider) GatherMetrics(r *http.Request, appSlug string) ([]common.Metrics, error) {
-	payload, err := github.ValidatePayload(r, nil)
+	contentType, _, err := mime.ParseMediaType(r.Header.Get("Content-Type"))
+	if err != nil {
+		return nil, err
+	}
+
+	payload, err := github.ValidatePayloadFromBody(contentType, r.Body, "", nil)
```

Comment:
so this is how we skip the payload signature validation, right?

`ValidatePayloadFromBody(contentType string, readable io.Reader, signature string, secretToken []byte) (payload []byte, err error)`

signature is empty and secretToken is nil and this skips the signature validation.

Reply:
exactly